File: app/src/main/java/com/example/rects/SettingCurrentGame.java
package com.example.rects;

public class SettingCurrentGame {

    private static boolean gameLoose = false;
    private static int currentNumLVL = 0;

    public static void setGameLoose(boolean gL) {
        gameLoose = gL;
    }

    public static boolean isGameLoose() {
        return gameLoose;
    }

    public static int getCurrentNumLVL() {
        return currentNumLVL;
    }

    public static void setCurrentNumLVL(int currentNumLVL) {

        if(currentNumLVL >=1){
            SettingCurrentGame.currentNumLVL = currentNumLVL;
        }
        else {
            SettingCurrentGame.currentNumLVL = 1;
        }
    }
}

File: app/src/main/java/com/example/rects/game/config/SettingCurrentGame.kt
package com.example.rects.game.config

object SettingCurrentGame {
    private var gameLoose = false
    private var currentNumLevel = 1

    @JvmStatic fun setGameLoose(value: Boolean) { gameLoose = value }
    @JvmStatic fun isGameLoose(): Boolean = gameLoose
    @JvmStatic fun getCurrentNumLVL(): Int = currentNumLevel
    @JvmStatic fun setCurrentNumLVL(value: Int) {
        currentNumLevel = value.coerceAtLeast(1)
    }
}
